### PR TITLE
[GENERAL] Rename keys and fix typos

### PR DESCRIPTION
**Pull request by iCultureBud**

### CODE REVIEW

File: entities/config.go [hunk 1]
```diff
@@ -12,29 +12,21 @@ type Client struct {
...
-	UserName string `toml:"userName"`
-	// FIXME: change to "ID"
-	Nonce    string `toml:"nonce"`
-	Registed bool   `toml:"registered"`
-}
-
-type ServerToml_t struct {
-	Server ServerConfig_t `toml:"server"`
+	Name       string `toml:"user_name"`
+	ID         string `toml:"id"`
```
Comment (DrinoSan) [inline entities/config.go:21]: This is still a UUID

File: entities/config.go [hunk 2]
```diff
@@ -12,29 +12,21 @@ type Client struct {
...
 type ServerAccount_t struct {
-	UserName string `toml:"userName"`
-	// FIXME: change to "ID"
-	Nonce    string `toml:"nonce"`
-	Registed bool   `toml:"registered"`
-}
-
-type ServerToml_t struct {
-	Server ServerConfig_t `toml:"server"`
+	Name       string `toml:"user_name"`
```
Comment (DrinoSan) [inline entities/config.go:20]: Ist camel case toml standard?

File: entities/config.go [hunk 3]
```diff
@@ -12,29 +12,21 @@ type Client struct {
 	ID    string `toml:"user_id"`
 }
 
-type ServerConfig_t struct {
-	MotherShipIP   string            `toml:"motherShipIP"`
-	MotherShipPort string            `toml:"motherShipPort"`
-	MotherShipSSL  bool              `toml:"motherShipSSL"`
+type Server struct {
```
Comment (DrinoSan) [inline entities/config.go:15]: Where can I set port or ip

File: server/server.go [hunk 4]
```diff
@@ -24,9 +24,10 @@ const file_suffix string = ".cwf"
 
 var filesDir string
 var port int
+var config entities.ServerConfig_t
```
Comment (DrinoSan) [inline server/server.go:27]: If like this then is it not possible to just create it here 
var config =  entities.ServerToml_t or something like that

File: server/server.go [hunk 5]
```diff
@@ -382,19 +380,19 @@ func (checker cwfChecker_t) ServeHTTP(writer http.ResponseWriter, req *http.Requ
 		return
 	}
 
-	// Check for user nonce.
-	cliUserName := req.Header.Get("Cwf-User-Name")
-	user, ok := ServerUsers[cliUserName]
+	// Check for user ID.
+	cliName := req.Header.Get("Cwf-User-Name")
```
Comment (DrinoSan) [inline server/server.go]: I think this name is misleading. CLI is something else

File: server/server.go [hunk 6]
```diff
@@ -382,19 +380,19 @@ func (checker cwfChecker_t) ServeHTTP(writer http.ResponseWriter, req *http.Requ
...
 		http.Error(writer, "User not found! Please register ", http.StatusForbidden)
 		return
 	}
 
-	// Check for user nonce.
-	userNonce := req.Header.Get("Cwf-User-Nonce")
-	if user.Nonce != userNonce {
-		http.Error(writer, "Wrong UUID", http.StatusForbidden)
+	// Check for user ID.
+	userID := req.Header.Get("Cwf-User-Id")
```
Comment (DrinoSan) [inline server/server.go:393]: I still believe this should be uuid

**Review by DrinoSan — CHANGES_REQUESTED** [review]

File: entities/config.go (hunk 3 above)
Comment (iCultureBud) [reply entities/config.go:15]: They both are set with flags when running the server

File: entities/config.go (hunk 1 above)
Comment (iCultureBud) [reply entities/config.go:21]: It's still an ID - so we can have other IDs in the future

File: entities/config.go (hunk 3 above)
Comment (DrinoSan) [reply entities/config.go:15]: If no flag provided then its localhost and 7878 if yes then okay. But I am not setting flags for local testing

File: entities/config.go (hunk 2 above)
Comment (iCultureBud) [reply entities/config.go:20]: There is no standard, but on toml.io they use snake case

File: server/server.go (hunk 5 above)
Comment (iCultureBud) [reply server/server.go]: ooouh yes - should be something more accurate

File: entities/config.go (hunk 3 above)
Comment (iCultureBud) [reply entities/config.go:15]: yep they both have default values

**Pull request merged**